src/Notification/Driver/NotificationDriverInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Notification\Driver;
+
+use Flarum\Notification\Blueprint\BlueprintInterface;
+use Flarum\User\User;
+
+interface NotificationDriverInterface
+{
+    /**
+     * Enqueues the job for this notification.
```

Comment:
Using the queue is an implementation detail. I think this should be revised to just say "Conditionally sends a notification to users, generally using a queue" or something like that.

Reply:
good point.